### PR TITLE
[ZH] Update `Beatmap Spotlights` and remove stale `outdated: true` tag from other translations

### PR DESCRIPTION
**Pull request by Kotoki1337**
- `Beatmap_Spotlights/Seasons` is all up-to-date but I don't know why they have outdated tags.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmap_Spotlights/zh.md [hunk 1]
```diff
@@ -137,8 +135,8 @@ outdated_since: 798269cfe9f0515e267a7f8a122b28fdff0aaf05
 
 1. 在一个赛季的时间中，谱面将由各个模式的策展人选择出来。
    - 策展人需要在公开讨论中就每个谱面达成共识。
-   - 各个游戏模式的负责人锁定最终决定，并在结论性讨论后巩固他们的选择。
-   - 选择的过程在各个游戏模式之间有所不同，并且会根据各个成员的需求进行调整。
+   - 模式负责人会锁定最终决定，并在结论性讨论后巩固他们的选择。
+   - 选择的过程在各个游戏模式之间有所不同，并且会根据成员的需求进行调整。
 2. 谱面的选择基于其独特性和卓越性。每个选定的谱面都应在游戏性，设计和美学方面出类拔萃。
 3. 最终所选的谱面可作为对整个 osu! 社区的推荐，并使用唯一的 *Spotlights* 标签标记。
```
Comment (YumeMuzi) [inline wiki/Beatmap_Spotlights/zh.md:141]:
```suggestion
3. 选定的谱面会推荐到整个 osu! 社区，并标上独一无二的 *Spotlights* 标签。
```

File: wiki/Beatmap_Spotlights/zh.md [hunk 2]
```diff
@@ -174,14 +172,36 @@ outdated_since: 798269cfe9f0515e267a7f8a122b28fdff0aaf05
 
 ## 历史
 
-该项目最初由 ::{ flag=US }:: [Cyclone](https://osu.ppy.sh/users/18589) 和 ::{ flag=AU }:: [peppy](https://osu.ppy.sh/users/2) 在 2009 年 11 月提出，原名 “Ranking Charts”。该项目旨在让 [BAT](/wiki/Modding/Beatmap_Appreciation_Team) 和 [MAT](/wiki/Modding/Mapping_Assistance_Team) 提名并投票选出展示为期一个月的最佳谱面。
+该项目最初由 ::{ flag=US }:: [Cyclone](https://osu.ppy.sh/users/18589) 和 ::{ flag=AU }:: [peppy](https://osu.ppy.sh/users/2) 在 2009 年 10 月[^charts-09-oct] [^charts-09-nov]提出，原名 “Ranking Charts”。该项目旨在让 [BAT](/wiki/Modding/Beatmap_Appreciation_Team) 和 [MAT](/wiki/Modding/Mapping_Assistance_Team) 提名并投票选出展示为期一个月或一年[^charts-10-jan]的最佳谱面。2011 年 9 月, osu!taiko 和 osu!catch 随之出现[^charts-11-per-mode]。
```
Comment (YumeMuzi) [inline wiki/Beatmap_Spotlights/zh.md:175]:
```suggestion
该项目最初由 ::{ flag=US }:: [Cyclone](https://osu.ppy.sh/users/18589) 和 ::{ flag=AU }:: [peppy](https://osu.ppy.sh/users/2) 在 2009 年 10 月[^charts-09-oct] [^charts-09-nov]提出，原名 “Ranking Charts”。该项目旨在让 [BAT](/wiki/Modding/Beatmap_Appreciation_Team) 和 [MAT](/wiki/Modding/Mapping_Assistance_Team) 提名并投票选出展示为期一个月或一年[^charts-10-jan]的最佳谱面。2011 年 9 月, osu!taiko 和 osu!catch 也启动了类似的项目[^charts-11-per-mode]。
```

File: wiki/Beatmap_Spotlights/zh.md [hunk 3]
```diff
@@ -174,14 +172,36 @@ outdated_since: 798269cfe9f0515e267a7f8a122b28fdff0aaf05
...
+2014 年 1 月，月赛浏览器被添加到 osu! 客户端当中[^charts-in-osu-14-jan]，随后的 2014 年 6 月，几位著名玩家被招募进入了 [CAT](/wiki/Beatmap_Spotlights/Chart_Assembly_Team)[^charts-cat-recruitment-14-jun]。从 2015 年 3 月开始，该项目从最初的提名和投票转变为让[知名社区成员逐个选择他们推荐的谱面集](https://osu.ppy.sh/home/news/2015-03-18-february-2015-monthly-ranking-charts-new-season)[^charts-curated-15-mar]。在2016年9月，提名系统已大体恢复[^charts-reverted-16-sep]，由 [QAT](/wiki/Modding/Quality_Assurance_Team) 负责选择最值得关注的谱面集。
+
+在 2017 年 3 月改名为 [Beatmap Spotlights](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights)[^charts-renamed-into-spotlights]。该系统本身保持了基本的连贯性，同时增加了诸如成就之类的奖励，并进一步增强了 Beatmap Spotlights 的显示效果。在对 QAT 进行内部检修期间，该项目的负责人已被重新分配给了 ::{ flag=HU }:: [Kurokami](https://osu.ppy.sh/users/260933) 并重新组建了一个基于社区的谱面选择团队。在 2018 年 11 月，月赛的频率被更改为季节性[^spotlights-seasonal]。2020 年 3 月，::{ flag=DE }:: [Loctav](https://osu.ppy.sh/users/71366) 随 Kurokami 重回项目负责人，将其重新设计成新的形状，并组建了新的名为 osu! 策展人的团队[^spotlights-reworked-20-june]。
```
Comment (YumeMuzi) [inline wiki/Beatmap_Spotlights/zh.md:183]:
```suggestion
在 2017 年 3 月改名为 [Beatmap Spotlights](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights)[^charts-renamed-into-spotlights]。该系统本身保持了基本的连贯性，同时增加了诸如成就之类的奖励，并进一步增强了 Beatmap Spotlights 的显示效果。在 QAT 休整期间， ::{ flag=HU }:: [Kurokami](https://osu.ppy.sh/users/260933) 重新任职为该项目的负责人，并重组了一个基于社区的谱面选择团队。2018 年 11 月，月赛的频率更改为季节性[^spotlights-seasonal]。2020 年 3 月，::{ flag=DE }:: [Loctav](https://osu.ppy.sh/users/71366) 和 Kurokami 重任项目负责人、重新设计项目结构、并新建了名为 osu! 策展人的团队[^spotlights-reworked-20-june]。
```

File: wiki/Beatmap_Spotlights/zh.md [hunk 4]
```diff
@@ -174,14 +172,36 @@ outdated_since: 798269cfe9f0515e267a7f8a122b28fdff0aaf05
...
 
-在 2017 年 3 月改名为 [Beatmap Spotlights](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights)。该系统本身保持了基本的连贯性，同时增加了诸如成就之类的奖励，并进一步增强了 Beatmap Spotlights 的显示效果。在对 QAT 进行内部检修期间，该项目的负责人已被重新分配给了 ::{ flag=HU }:: [Kurokami](https://osu.ppy.sh/users/260933) 并重新组建了一个基于社区的谱面选择团队。在 2018 年 11 月，月赛的频率被更改为[季节性](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018)。2020 年 3 月，::{ flag=DE }:: [Loctav](https://osu.ppy.sh/users/71366) 随 Kurokami 重回项目负责人，将其重新设计成新的形状，并组建了新的名为 osu! 策展人的团队。
+2014 年 1 月，月赛浏览器被添加到 osu! 客户端当中[^charts-in-osu-14-jan]，随后的 2014 年 6 月，几位著名玩家被招募进入了 [CAT](/wiki/Beatmap_Spotlights/Chart_Assembly_Team)[^charts-cat-recruitment-14-jun]。从 2015 年 3 月开始，该项目从最初的提名和投票转变为让[知名社区成员逐个选择他们推荐的谱面集](https://osu.ppy.sh/home/news/2015-03-18-february-2015-monthly-ranking-charts-new-season)[^charts-curated-15-mar]。在2016年9月，提名系统已大体恢复[^charts-reverted-16-sep]，由 [QAT](/wiki/Modding/Quality_Assurance_Team) 负责选择最值得关注的谱面集。
```
Comment (YumeMuzi) [inline wiki/Beatmap_Spotlights/zh.md:181]:
```suggestion
2014 年 1 月，月赛浏览器被添加到 osu! 客户端当中[^charts-in-osu-14-jan]，随后的同年 6 月，几位著名玩家被招募进入了 [CAT](/wiki/Beatmap_Spotlights/Chart_Assembly_Team)[^charts-cat-recruitment-14-jun]。从 2015 年 3 月开始，该项目从最初的提名和投票，转变为让[知名社区成员逐个选择他们推荐的谱面](https://osu.ppy.sh/home/news/2015-03-18-february-2015-monthly-ranking-charts-new-season)[^charts-curated-15-mar]。在 2016 年 9 月，提名系统已大体恢复[^charts-reverted-16-sep]。此时，[QAT](/wiki/Modding/Quality_Assurance_Team) 将负责选择最值得关注的谱面。
```

**Review by YumeMuzi — CHANGES_REQUESTED** [review]
Not sure you should continue to modify the rest paragraph

File: wiki/Beatmap_Spotlights/zh.md [hunk 5]
```diff
@@ -137,14 +135,14 @@ outdated_since: 798269cfe9f0515e267a7f8a122b28fdff0aaf05
...
 2. 谱面的选择基于其独特性和卓越性。每个选定的谱面都应在游戏性，设计和美学方面出类拔萃。
-3. 最终所选的谱面可作为对整个 osu! 社区的推荐，并使用唯一的 *Spotlights* 标签标记。
+3. 选定的谱面会推荐到整个 osu! 社区，并标上独一无二的 *Spotlights* 标签。
 4. 为了履行向整个社区推荐优秀谱面的职责，谱面应涵盖特定范围的 Hard，Insane 和 Expert 难度。
-   - 在所有谱面中，应该有 6 张谱面为 Hard 难度内。
-   - 在所有谱面中，应该有 9 张谱面为 Insane 难度内。
-   - 在所有谱面中，应该有 12 张谱面为 Expert 难度内。
+   - 在所有谱面中，应有 6 张谱面在 Hard 难度内。
+   - 在所有谱面中，应有 9 张谱面在 Insane 难度内。
+   - 在所有谱面中，应有 12 张谱面在 Expert 难度内。
```
Comment (YumeMuzi) [inline wiki/Beatmap_Spotlights/zh.md:145]:
```suggestion
   - 所有谱面中，应至少包含 6 个 Hard 难度。
   - 所有谱面中，应至少包含 9 个 Insane 难度。
   - 所有谱面中，应至少包含 12 个 Expert 难度。
```

File: wiki/Beatmap_Spotlights/zh.md [hunk 6]
```diff
@@ -174,14 +172,36 @@ outdated_since: 798269cfe9f0515e267a7f8a122b28fdff0aaf05
...
+[^charts-cat-recruitment-14-jun]: [(Restricted) Forum thread by Loctav (2014-06-16) "\[IMPORTANT\] Recruiting NewCATs"](https://osu.ppy.sh/community/forums/topics/218032)
+[^charts-curated-15-mar]: [News post by Loctav (2015-03-18) "February 2015 Monthly Ranking Charts - New Season!"](https://osu.ppy.sh/home/news/2015-03-18-february-2015-monthly-ranking-charts-new-season)
+[^charts-reverted-16-sep]: [News post by OnosakiHito (2016-09-17) "July 2016 Ranking Charts - Changes"](https://osu.ppy.sh/home/news/2016-09-17-july-2016-ranking-charts-changes)
+
+[^charts-renamed-into-spotlights]: [News post by OnosakiHito (2017-03-18) "Introducing to you: Spotlights"](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights)
+[^spotlights-seasonal]: [News post by Kurokami (2018-11-01) "Seasonal Spotlights: Summer 2018"](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018)
+[^spotlights-reworked-20-june]: [Forum thread by Loctav (2020-07-06) "Beatmap Spotlights (Summer 2020) - Discussion Thread"](https://osu.ppy.sh/community/forums/topics/1101170)
```
Comment (TicClick) [inline wiki/Beatmap_Spotlights/zh.md:207]: references need to be translated, too (except for the thread and article titles). example: https://github.com/ppy/osu-wiki/pull/8265

**Review by TicClick — CHANGES_REQUESTED** [review]

**Review by YumeMuzi — APPROVED** [review]
seems fine

**Review by TicClick — APPROVED** [review]

**Pull request merged**